<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CacheInspector</class>
 <widget class="QWidget" name="CacheInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Select plot view</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="crbutton">
          <property name="text">
           <string>Cache requests</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">selectedPlot</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="crrbutton">
          <property name="text">
           <string>Cache requests (ratio)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">selectedPlot</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="trbutton">
          <property name="text">
           <string>Temporal requests</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">selectedPlot</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="trrbutton">
          <property name="text">
           <string>Temporal requests (ratio)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">selectedPlot</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="plotview">
     <widget class="QCustomPlot" name="cacheRequestPlot"/>
     <widget class="QCustomPlot" name="cacheRequestRatioPlot"/>
     <widget class="QCustomPlot" name="temporalPlot"/>
     <widget class="QCustomPlot" name="temporalRatioPlot"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="selectedPlot"/>
 </buttongroups>
</ui>
